README.md

```diff
@@ -1 +1,27 @@
-# Lipohypertrophy Classification Model Deployment
+# Lipohypertrophy Model Deployment
+
```

Comment:
Maybe add a link to the deployed streamlit app?

To see the app in action, click [here](https://share.streamlit.io/xudongyang2/lipo_deployment/demo_v2.py).

Reply:
Thanks @jraza19 ! Addressed.